components/knative-function-controller/Makefile

```diff
@@ -72,7 +61,7 @@ deploy: manifests install
 
 # Build the docker image
 .PHONY: docker-build
-docker-build: resolve pull-licenses manifests
+docker-build: resolve pull-licenses test manifests
```

Comment:
Do we need to execute test before `docker-build` when building images locally? Tests take sometime and when I want to build an image I usually don't want to run tests. One can still run `make test`. 

Is this used in CI? Then I would agree to have `test` as part of `docker-build`. Just an idea :)